HydraIRC/version.h

```diff
@@ -66,4 +66,4 @@
 #endif
 */
 
-#define AUTHORSTRING "Dominic Clifton aka Hydra"
+#define AUTHORSTRING "Dominic Clifton aka Hydra\n Upgrades and newer OS and VS support by gdwnldsKSC"
```

Comment:
Understand you'd like some attribute, but not via this.  Feel free to add additional strings and make them appear on the Help/About dialog.